### PR TITLE
ENH Make ElementalContentExtension MyPage a required field

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/pull/1178

### CODE REVIEW

File: code/TestRegistryPage.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 use SilverStripe\Security\DefaultAdminService;
 use SilverStripe\Security\Member;
 
-if (!class_exists(RegistryPage::class)) {
+if (class_exists(RegistryPage::class)) {
```
Comment (emteknetnz) [inline code/TestRegistryPage.php:9]: This was clearly wrong, it broke my first attempt at doing a shared ci run

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Please update the commit message - seems incorrect?

**Comment by emteknetnz** [comment]
Done

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when main PR is ready to merge

**Pull request merged**